src/main/resources/application.yaml

```diff
@@ -76,14 +76,15 @@ hapi:
     #    install_transitive_ig_dependencies: true
     ### tells the server whether to attempt to load IG resources that are already present
     #    reload_existing_implementationGuides : false
-    #    implementationguides:
+    implementationguides:
     ###    example from registry (packages.fhir.org)
-    #      swiss:
-    #        name: swiss.mednet.fhir
-    #        version: 0.8.0
+      swiss:
+        name: swiss.mednet.fhir
+        version: 0.8.0
+        reloadExisting : false
```

Comment:
@jkiddo Why is swiss.mednet.fhir changing from being left out to being included in the application?

Reply:
No apparent reason - it was for test and I forgot to remove it again.

Reply:
was commented out